src/CLU/Microsoft.CLU/CommandLineParser/UnixCommandLineParser.cs

```diff
@@ -121,11 +121,11 @@ public bool Parse(ICommandBinder commandBinder, string[] arguments)
                 }
                 else
                 {
-                    if (position == 0 && 
-                        commandBinder.SupportsAutomaticHelp && 
+                    if (position == 0 &&
+                        commandBinder.SupportsAutomaticHelp &&
                         arg.Equals("help", System.StringComparison.OrdinalIgnoreCase))
```

Comment:
I wonder if we should abstract this into a method for determining whether we have a help argument.  For example, ```help``` should be able to appear in any position and ```-h``` should be equivalent to ```--help```